src/Services/MediaLibrary/Glide.php

```diff
@@ -273,4 +269,29 @@ protected function getFocalPointCrop($crop_params, $width, $height)
 
         return [];
     }
+
+    /**
+     * @param string $id 
+     * @return string
+     */
+    private function getOriginalMediaUrl($id)
+    {
+        if (!Str::endsWith($id, $this->config->get('twill.glide.original_media_for_extensions'))) {
+            return false;
+        }
+
+        $libraryDisk = $this->config->get('twill.media_library.disk');
+        $endpointType = $this->config->get('twill.media_library.endpoint_type');
+        $localPath = $this->config->get('twill.media_library.local_path');
+        $appUrl = $this->config->get('app.url');
+
+        switch ($endpointType) {
+            case 'local':
+                return $appUrl . '/storage/' . $localPath;
```

Comment:
Thank you so much @Keimeno and @pboi20 for moving this forward. This line is the only thing I'm a bit on the fence about. Could we use the Flysystem disk here instead to get this url?